source/fundamentals/crud/write-operations/modify.txt

```diff
@@ -13,11 +13,23 @@ Modify Documents
 Overview
 --------
 
-You can modify documents in a MongoDB collection using either **update**
-or **replace** operations. Update operations mutate
+You can modify documents in a MongoDB collection using **update**
+and **replace** operations. Update operations change
 specified fields in one or more documents and leave other fields and values
-unchanged. Replace operations remove all existing fields in one or more
-documents and substitute them with specified fields and values.
+unchanged. Replace operations remove all fields except the ``_id`` field
+in an existing document and substitute them with specified fields and values.
```

Comment:
Suggestion:
I think if the prior suggestion is taken, this sentence could provide a parallel structure to make it easier to compare the two.

```suggestion
unchanged. Replace operations substitute the document fields and values with the specified replacement document while keeping the ``_id`` value.
```